DAL/Repositories/Interfaces/IOrderItemRepository.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace DAL.Repositories.Interfaces;
+
+public interface IOrderItemRepository
+{
```

Comment:
you need to implement IBaseRepository like IOrderItemRepository : IBaseRepository